### DISCUSSION TITLE
nodepay.ai

### DISCUSSION THREAD

**Discussion by tannguyen06** [engageub/InternetIncome#131] [General]
Can you add this website? It is similar to getgrass.
nodepay.ai

**Comment by engageub** [comment]
Hi,
Thank you for letting me know about this.
An issue has been created to update the existing docker container for nodepay. Once it is resolved, changes will be pushed. 

https://github.com/doubleTroub1e/nodepayai/issues/4


Thank you

**Reply by engageub** [reply]
Hi,
Before the issues are resolved, if you would like to use any app which is related to installing browser extensions,
you can use [test](https://github.com/engageub/InternetIncome/tree/test) branch and set CUSTOM_CHROME to true in properties.conf file.
You will get a browser with new profile, where you can install any extensions as regular browser.
You can create multiple instances of chrome browser with different proxies, except the credentials and installing the extensions are manual. 

Thank you

**Reply by tannguyen06** [reply]
Thank you, I will try it that way.

**Reply by superit63** [reply]
Hi,
In my opion, Nodepay is likely a scam project. Btw, if you want to try, you can use python code to get direct to the websocket instead of using Chrome profile.
Thanks

**Reply by engageub** [reply]
Yes, there is already a docker image published which uses websocket, but I am sceptical as I have few concerns.
Please see the following issue for reference.
https://github.com/aston668334/nodepay-socks-python/issues/4
